html/php-challenge/index.php

```diff
@@ -70,7 +70,46 @@ function makeLink($value)
 {
     return mb_ereg_replace("(https?)(://[[:alnum:]\+\$\;\?\.%,!#~*/:@&=_-]+)", '<a href="\1\2">\1\2</a>', $value);
 }
+
+
+
+//いいねを押したかどうか
+function isPushLike(){
+    if(isset($_REQUEST['like'])){
+        return true;
+    }
+}
```

Comment:
false を返却しないのは不健全なのでこうしましょう。

```suggestion
function isPushLike()
{
    return isset($_REQUEST['like']);
}
```